src/firefly/js/visualize/projection/TpvProjection.js

```diff
@@ -0,0 +1,257 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+
+import {DtoR,RtoD, acosd,asind,atan2d,atand,sind,cosd,tand} from './ProjectionUtil.js';
+import {makeProjectionPt,makeImagePt} from '../Point.js';
+
```

Comment:
File seems fine to me.  There is some stuff that is not used from ProjectionUtil.js
and some warning about let and const that could be fixed.